tests/run/for_in_iter.py

```diff
@@ -143,8 +143,7 @@ def for_in_gen(N):
     >>> for_in_pyiter(for_in_gen(10))
     [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
     """
-    for i in range(N):
-        yield i
+    yield from range(N)
```

Comment:
This looks like a change to what's being tested.

We might well want to test `yield from` too. But the intent was definitely to test `yield` in a loop